
[2017-05-21 02:06:22,876] - UploadSpss.py [Line:93] - [INFO]-[thread:139876461602560]-[process:72339] - [data write database success !!!]
[2017-05-21 02:06:23,154] - UploadSpss.py [Line:164] - [ERROR]-[thread:139876461602560]-[process:72339] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:16:40,995] - UploadSpss.py [Line:94] - [INFO]-[thread:139631183640320]-[process:72517] - [data write database success !!!]
[2017-05-21 02:16:41,224] - UploadSpss.py [Line:165] - [ERROR]-[thread:139631183640320]-[process:72517] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:17:02,586] - UploadSpss.py [Line:94] - [INFO]-[thread:140223765354240]-[process:72537] - [data write database success !!!]
[2017-05-21 02:17:02,924] - UploadSpss.py [Line:165] - [ERROR]-[thread:140223765354240]-[process:72537] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:18:39,920] - UploadSpss.py [Line:95] - [INFO]-[thread:140033776211712]-[process:72594] - [data write database success !!!]
[2017-05-21 02:18:40,215] - UploadSpss.py [Line:166] - [ERROR]-[thread:140033776211712]-[process:72594] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:19:13,896] - UploadSpss.py [Line:95] - [INFO]-[thread:139624166463232]-[process:72621] - [data write database success !!!]
[2017-05-21 02:19:14,194] - UploadSpss.py [Line:166] - [ERROR]-[thread:139624166463232]-[process:72621] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:23:22,528] - UploadSpss.py [Line:96] - [INFO]-[thread:139777778403072]-[process:72671] - [data write database success !!!]
[2017-05-21 02:23:22,759] - UploadSpss.py [Line:167] - [ERROR]-[thread:139777778403072]-[process:72671] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:25:57,778] - UploadSpss.py [Line:96] - [INFO]-[thread:140388313650944]-[process:72742] - [data write database success !!!]
[2017-05-21 02:25:58,096] - UploadSpss.py [Line:168] - [ERROR]-[thread:140388313650944]-[process:72742] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:28:26,825] - UploadSpss.py [Line:96] - [INFO]-[thread:140144118621952]-[process:72805] - [data write database success !!!]
[2017-05-21 02:28:27,139] - UploadSpss.py [Line:259] - [ERROR]-[thread:140144118621952]-[process:72805] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:29:36,553] - UploadSpss.py [Line:96] - [INFO]-[thread:140145567106816]-[process:72840] - [data write database success !!!]
[2017-05-21 02:29:36,828] - proj_ques_handlers.py [Line:248] - [ERROR]-[thread:140145567106816]-[process:72840] - [(1264, "Out of range value for column 'VarWidth' at row 1")]
[2017-05-21 02:30:35,868] - UploadSpss.py [Line:96] - [INFO]-[thread:140350626158336]-[process:72882] - [data write database success !!!]
[2017-05-21 02:36:07,626] - UploadSpss.py [Line:96] - [INFO]-[thread:140577142945536]-[process:73011] - [data write database success !!!]
[2017-05-21 02:41:46,299] - UploadSpss.py [Line:96] - [INFO]-[thread:140283547592448]-[process:73096] - [data write database success !!!]
[2017-05-21 02:48:49,674] - UploadSpss.py [Line:96] - [INFO]-[thread:140635215152896]-[process:73201] - [data write database success !!!]
[2017-05-21 02:48:49,922] - UploadSpss.py [Line:167] - [INFO]-[thread:140635215152896]-[process:73201] - [write spss data head information successful!]
[2017-05-21 02:49:37,936] - proj_ques_handlers.py [Line:248] - [ERROR]-[thread:140635215152896]-[process:73201] - ['FileSpss']
[2017-05-21 03:36:45,542] - UploadSpss.py [Line:96] - [INFO]-[thread:140635215152896]-[process:73202] - [data write database success !!!]
[2017-05-21 03:36:45,808] - UploadSpss.py [Line:167] - [INFO]-[thread:140635215152896]-[process:73202] - [write spss data head information successful!]
[2017-05-21 06:50:58,482] - UploadSpss.py [Line:92] - [INFO]-[thread:140039762339584]-[process:75640] - [data write database success !!!]
[2017-05-21 06:50:58,889] - UploadSpss.py [Line:163] - [INFO]-[thread:140039762339584]-[process:75640] - [write spss data head information successful!]
[2017-05-22 01:07:22,126] - proj_ques_handlers.py [Line:306] - [ERROR]-[thread:139628069598976]-[process:85300] - [No axis named { for object type <class 'pandas.core.frame.DataFrame'>]
[2017-05-22 01:09:25,800] - proj_ques_handlers.py [Line:306] - [ERROR]-[thread:139628069598976]-[process:85299] - [No axis named { for object type <class 'pandas.core.frame.DataFrame'>]
[2017-05-22 01:10:17,886] - proj_ques_handlers.py [Line:307] - [ERROR]-[thread:140041763559168]-[process:85412] - [No axis named { for object type <class 'pandas.core.frame.DataFrame'>]
[2017-05-22 01:13:28,934] - proj_ques_handlers.py [Line:306] - [ERROR]-[thread:140707658610432]-[process:85497] - [invalid literal for int() with base 10: '4.0']
[2017-05-22 01:14:36,372] - proj_ques_handlers.py [Line:307] - [ERROR]-[thread:139829773768448]-[process:85533] - [invalid literal for int() with base 10: '1.0']
[2017-05-22 01:14:52,735] - proj_ques_handlers.py [Line:308] - [ERROR]-[thread:139705842525952]-[process:85552] - [invalid literal for int() with base 10: '1.0']
[2017-05-22 01:26:43,323] - proj_ques_handlers.py [Line:305] - [ERROR]-[thread:140632462677760]-[process:85759] - ['DataFrame' object has no attribute 'value_counts']
[2017-05-22 01:39:53,204] - proj_ques_handlers.py [Line:305] - [ERROR]-[thread:139806115092224]-[process:85981] - [keys must be a string]
[2017-05-22 08:22:19,035] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140531170195200]-[process:87498] - [list index out of range]
[2017-05-22 08:26:11,683] - proj_ques_handlers.py [Line:368] - [ERROR]-[thread:139639774983936]-[process:90430] - ['ID']
[2017-05-22 08:29:04,040] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90470] - [list index out of range]
[2017-05-22 08:32:29,756] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90470] - [list index out of range]
[2017-05-22 08:32:41,994] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90470] - ['ID']
[2017-05-22 08:34:13,648] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90470] - ['StartTime']
[2017-05-22 08:35:14,799] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90470] - ['ID']
[2017-05-22 08:35:52,305] - proj_ques_handlers.py [Line:365] - [ERROR]-[thread:140487015991040]-[process:90471] - ['ID']
